addons/udes_stock/views/stock_picking.xml

```diff
@@ -101,6 +101,11 @@
                 <attribute name="invisible">1</attribute>
             </xpath>
 
+            <!-- Add button to show all fields, by default hidden fields will not be shown -->
+            <xpath expr="//button[@name='action_confirm']" position="before">
+                <button name="view_more_info" string="View More" type="object" class="oe_highlight"/>
```

Comment:
Can we call this `Detailed View`? Think it reads a bit better but might be something for product. :)

Also I suggest removing `class="oe_highlight"` - it is a difficult one to judge but it isn't a button that affects the flow of any model.

Reply:
Made the changes in both MR-s